scripts/consolidate-memory.py

```
@@ -0,0 +1,492 @@
+    @property
+    def count(self) -> int:
+        return len(self.items)
+
+    @property
+    def dates(self) -> list[date]:
+        return sorted(set(item.entry_date for item in self.items))
+
+
+def get_memory_dir() -> Path:
+    """Find memory directory."""
+    for parent in [Path.cwd()] + list(Path.cwd().parents):
```

Ruff (0.14.14)

[warning] Consider `[Path.cwd(), *list(Path.cwd().parents)]` instead of concatenation

Replace with `[Path.cwd(), *list(Path.cwd().parents)]`

(RUF005)